universitat-bremen-institut-fur-politikwissenschaft.csl

```diff
@@ -67,24 +67,64 @@
   </macro>
   <macro name="access">
     <choose>
-      <if type="webpage article-journal article-magazine article-newspaper article speech" variable="URL" match="all">
-        <group>
-          <text value="Text abrufbar unter:" prefix=" " suffix=" "/>
-          <text variable="URL"/>
-          <group prefix=" (" suffix=")">
-            <text value="Zugriff am" suffix=" "/>
-            <date variable="accessed">
-              <date-part name="day" form="numeric" suffix="."/>
-              <date-part name="month" form="numeric" suffix="."/>
-              <date-part name="year" form="long"/>
-            </date>
-          </group>
-        </group>
+      <if variable="URL">
+        <choose>
+          <if type="article-journal" variable="DOI" match="all">
+            <choose>
+              <if variable="volume" match="none">
+                <group>
+                  <text value="Text abrufbar unter:" prefix=" " suffix=" "/>
+                  <text variable="URL"/>
+                </group>
+              </if>
+            </choose>
+          </if>
+          <else-if>
```

Comment:
you can't have an `<else-if>` without an actual condition. I think you just want `<else-if type="article-journal"/>` here and then get rid of the extra text you have in 83+84